Library/Formula/tidy-html5.rb

```diff
@@ -0,0 +1,14 @@
+class TidyHtml5 < Formula
+  homepage "http://www.html-tidy.org/"
+  url "https://github.com/htacg/tidy-html5/archive/tag-4.9.25.tar.gz"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV.deparallelize
```

Comment:
I just added this defensively not really knowing if it's necessary... it seems it'll do fine with parallel builds.